tests/v2/test_waku_rln_relay_onchain.nim

```diff
@@ -533,3 +560,17 @@ procSuite "Waku-rln-relay":
 
     await node.stop()
     await node2.stop()
+
+  ################################
+  ## Terminating/removing Ganache
+  ################################
```

Comment:
Same here. Can you wrap the ganache kill and uninstall logic in a `proc` (e.g., `proc cleanupGanache()`) and call it here?